src/components/generics/vacation-test-forms/employeeVacationRequests.tsx

```diff
@@ -9,23 +9,24 @@ import { CalendarPickerView } from "@mui/x-date-pickers";
 import strings from "localization/strings";
 import DateRangePicker from "../date-range-picker/date-range-picker";
 import { FilterScopes } from "types";
+import { VacationRequestStatus, VacationType } from "generated/client";
 
 /**
  * interface type request
  */
 interface Request {
   request: {
     id: number;
-    vacationType: string;
-    comment: string;
+    vacationType: VacationType | string;
```

Comment:
Again, as VacationType is an enum, and therefore ultimately a string, can we just type this as VacationType?